pkg/combustion/templates/31-elemental-register.sh.tpl

```diff
@@ -20,16 +37,17 @@ After=time-sync.target
 
```

Comment:
Does this service require the other being successful? if so, a Wants directive here won't hurt

Reply:
I don't think this matters at all at this stage of the build. I'd prefer to leave it aligned with the Elemental team's systemd unit files.